diff --git a/src/foundry/client/pixi/placeables.d.ts b/src/foundry/client/pixi/placeables.d.ts
index 08e873ddaaaeb47aac5dd9b9fb6b9bdeace2a2d0..254c6e8ff58cd20cdc74b0e0686b48d6817041db 100644
--- a/src/foundry/client/pixi/placeables.d.ts
+++ b/src/foundry/client/pixi/placeables.d.ts
@@ -7,7 +7,13 @@ import {
 } from '../../../types/helperTypes';
 import EmbeddedCollection from '../../common/abstract/embedded-collection.mjs';
 
-type ConcretePlaceableOrPlaceableObject<T> = T extends PlaceableObject ? T : PlaceableObject;
+// XXX Gross hack to get around "TS2589: Type instantiation is excessively deep
+// and possibly infinite." in TS @5.4.2.
+//
+// original version:
+// type ConcretePlaceableOrPlaceableObject<T> = T extends PlaceableObject ? T :
+// PlaceableObject;
+type ConcretePlaceableOrPlaceableObject<T> = PlaceableObject;
 
 declare global {
   /**
